playwright-e2e/dss/pages/atcp/atcp-dashboard-page.ts

```diff
@@ -26,7 +26,20 @@ export default class AtcpDashboardPage extends AtcpPageBase {
 
   async expandTable(): Promise<void> {
     await waitForNoSpinner(this.page);
-    return this.getTable().headerLocator().waitFor({ state: 'visible', timeout: 5000 })
-      .catch(async () => await this.page.locator('.participant-expandable button.participant-expandable__control').click());
+    const expand = this.page.locator('.participant-expandable button.participant-expandable__control');
```

Comment:
Don't know why, table expanded then quickly collapsed. Adding checks to ensure table remains open.